backend/src/main/kotlin/cz/loono/backend/db/model/Account.kt

```diff
@@ -38,7 +38,11 @@ data class Account(
 
     @OneToMany(orphanRemoval = true, cascade = [CascadeType.ALL], mappedBy = "account")
     @Column(nullable = false, updatable = true, insertable = true)
-    val examinationRecords: List<ExaminationRecord> = mutableListOf()
+    val examinationRecords: List<ExaminationRecord> = mutableListOf(),
+
+    @OneToMany(orphanRemoval = true, cascade = [CascadeType.ALL], mappedBy = "account")
```

Comment:
Vážne to chceme? Za mňa super nebezpečná vec, ktorá zmaže polku DB pri malej chybe.

Reply:
Tak na todle nemam silny nazor, je to problem, ze kdyz odstranime account chybne, ze nam nezustanou zadne badges nebo je vetsi problem, ze my jsme chybne odstranili account? Jinak `examinations` maji stejny mapping, tak mozna bychom se meli rozdhnout application-wise jaky pristup zvolime, pro badges to jako problem nevidim, asi se nic nestane, kdyz zmizeji badge u uzivatele

Reply:
Za mňa zjednotiť tak, že sa nemaže. Problém je v orphanRemoval = true, cascade = [CascadeType.ALL]. Badge budú mať to isté, a zmazaním jedného objektu sa zmaže celý strom. Už som videl, ako presne táto chyba zmazala tretinu DB. Radšej budem, ak nám ostanú v DB orphan. Dajú sa detekovať a v najhoršom ručne zmazať.
@d35h , @rposkocil - môžeme to dať ako pravidlo? Nepožívať orphanRemoval = true? Ak áno, začnem to spisovať na Wiki.

Reply:
Z my strany neni problem, opravim to pro badges

Reply:
Super. Prosím ešte task na opravu všade. Ďakujem.

Reply:
Done in [d56dcf3](https://github.com/cesko-digital/loono-be/pull/31/commits/d56dcf34792927cae9729a663d7c834941b741ba)